openlibrary/core/bookshelves.py

```diff
@@ -230,6 +321,9 @@ def get_users_logged_books(
         from openlibrary.plugins.worksearch.code import run_solr_query
         from openlibrary.plugins.worksearch.schemes.works import WorkSearchScheme
 
+        logger.info("BOOKSHELF: %s" % bookshelf_id)
```

Comment:
Don't miss these debug log lines!

Reply:
They have been executed, as an example to the others. I suspect the debuggers will know better than to show up in future PRs.